app/controllers/delivery/book_van.js

```diff
@@ -51,9 +51,9 @@ export default addressDetails.extend({
 
       var order = controller.store.createRecord('gogovan_order', requestProperties);
       order.set('delivery', delivery);
-
       new AjaxPromise("/gogovan_orders/calculate_price", "POST", controller.get('session.authToken'), requestProperties).then(function(data) {
           order.set('baseFee', data['base']);
+          loadingView.destroy();
           controller.transitionToRoute('delivery.confirm_van', {queryParams: {placeOrder: true}});
```

Comment:
@sunil-sharma what happens if calculate_price fails? Say if GGV API or our API is down? Does an error fire? What happens to loading screen in this case?